tools/pythonpkg/tests/fast/pandas/test_df_object_resolution.py

```diff
@@ -376,6 +376,21 @@ def test_integer_conversion_fail(self, duckdb_cursor):
         double_dtype = np.dtype('object')
         assert isinstance(converted_col['0'].dtype, double_dtype.__class__) == True
 
+    # numpy.datetime64 is just a wrapper around a datetime.datetime object, because reasons..
+    def test_numpy_datetime(self):
+        numpy = pytest.importorskip("numpy")
+
+        standard_vector_size = 2048
```

Comment:
Maybe this should be a shared variable somewhere?

Reply:
I've turned it into a static method on DuckDB and added it to the duckdb python module as `__standard_vector_size__`